doc/reference/hv-make-options.rst

```diff
@@ -0,0 +1,200 @@
+.. _hypervisor-make-options:
+
+Hypervisor Makefile Options
+###########################
+
+The ACRN hypervisor source code provides a makefile to build the ACRN
+hypervisor binary and associated components.
+
+Assuming that you are at the top level of the ``acrn-hypervisor`` directory,
+you can run the ``make`` command to start the build. See
+:ref:`acrn_configuration_tool` for information about required input files.
+
+Build Options
```

Comment:
I would split this section into two sections, one for the `Build Options` and one for the `Makefile Targets`. The third one below (i.e. `Makefile Targets for Configuration`) is good and should stay as it is.

* In the `Build Options`: list specific options: `BOARD`, `SCENARIO`, `RELEASE`. I would also add `ASL_COMPILER` to this list. It can be used in case you want to use a `iasl` compiler that is *not* in the `/usr/sbin` path that we use by default. E.g.: `make BOARD=nuc7i7dnb SCENARIO=industry ASL_COMPILER=/usr/local/bin/iasl`
* In the `Makefile Targets`: list the most relevant targets, i.e.: `hypervisor`, `devicemodel`, `tools`, `life_mngr`, `doc` and `targz-pkg`
   * `doc: Build the project's HTML documentation (using Sphinx), output to the ``build/doc`` folder.`
   * `life_mngr: Build the "Lifecycle Manager" daemon that runs in the User VM to manage power state transitions (S5)`
   * `targz-pkg: Create a compressed tarball (``acrn-$(FULL_VERSION).tar.gz``) in the build folder (default: ``build``) with all the build artefacts`

Reply:
Instead of making a separate section, I just made a separate table for the targets. I felt the examples looked better within the existing section.